workspaces/ui/src/components/navigation/Navbar.js

```diff
@@ -110,6 +110,22 @@ export default function Navbar(props) {
             <IconButton className={classes.navButton}>{i.icon}</IconButton>
           </LightTooltip>
         ))}
+        <div style={{flex: 1}}/>
+        <div>
+          <LightTooltip
+            // key={i.link}
+            title={"Optic on Discord - ask for help, request features, contribute something :)"}
+            component={'a'}
+            target="_blank"
+            href={'https://discord.gg/t9hADkuYjP'}
```

Comment:
should this be using our backend redirect?

Reply:
not deployed yet, will eventually update everywhere